docs/src/DataStructures/ApplicationContext.md

```diff
@@ -0,0 +1,63 @@
+# Application Context
+
+## Schema
+
+An application context credential MUST be a W3C Verifiable Credential document containing information about the application requesting user sign-in.
+
+The `type` array for the credential should begin with the string `"ApplicationContextCredential"`.
+
+The `credentialSchema` field should contain an object with the following fields:
+
+- `type`: MUST be `JsonSchema`
+- `id`: MUST be the URL of the published application context credential schema. The latest version is [`https://schemas.frequencyaccess.com/ApplicationContextCredential/bciqe2bsnuaqg7zy3gqjmwha2q5h2bybvr6log2jsb5kjn2hos6irrlq.json`](https://schemas.frequencyaccess.com/ApplicationContextCredential/bciqe2bsnuaqg7zy3gqjmwha2q5h2bybvr6log2jsb5kjn2hos6irrlq.json).
+
+The `credentialSubject` field should contain an object with the following fields:
+
+- `id`: Your provider identifier, expressed as a DID.
+- `application`: A JSON object containing the following keys:
+  - `name`: A map of one or more language tags to human-readable string values. Language tags should follow [BCP-47/RFC-5646](https://www.rfc-editor.org/rfc/rfc5646.html) (as used in the HTTP `Content-Language` header). A content language key of `"*"` indicates a wildcard or default value, as in HTTP.
+  - `logo`: A map of one or more language tags (as above) to objects containing a `url` property. The URL should resolve to an image in the PNG format. The recommended maximum image width for optimal client compatibility is 200 pixels.
```

Comment:
We should just spec a specific width and height. That would clarify any issues with ratio and such as well.

Reply:
This came from the product spec, will raise with CL.